src/backend/controllers/letterControllers.ts

```diff
@@ -110,6 +110,7 @@ export async function assignLetter({
     assignedResponderFullName,
     status,
     replyStatus,
+    replyStatusTimestamp
```

Comment:
This is a good idea to refactor the code, but should we do it under this issue or put it as another task? This could also apply to other types (bookings, etc.) that we haven't checked.

Reply:
Now I think it is best to just separate it from this merge request and do it later if needed. It is not directly related to this feature. And it is quite big change anyways, one reason is also that as a separate task/MR the changes will not clutter what is happening here.